components/team_sidebar/team_sidebar.tsx

```diff
@@ -3,19 +3,41 @@
 
 import React from 'react';
 
-type Props = {
+import {Team} from 'mattermost-redux/types/teams';
+
+import TeamButton from './team_button';
 
+type Props = {
+    teams: Team[];
 };
 
 type State = {
 
 };
 
 export default class TeamSidebar extends React.PureComponent<Props, State> {
+    renderTeam = (team: Team) => {
+        return (
+            <TeamButton team={team}/>
+        );
+    }
+
     render() {
+        const {teams} = this.props;
+        const renderedTeams = teams.map(this.renderTeam);
+
+        // TODO temp
+        const root = document.querySelector('#root');
+        if (teams.length <= 1) {
+            root!.classList.remove('multi-teams');
+            return null;
+        }
+        root!.classList.add('multi-teams');
```

Comment:
Maybe we could make a ticket to see what this class does, so we can remove it if possible?

Reply:
This class just moved the channel sidebar over when you have multiple teams. It will likely be reworked with the stuff @bradjcoughlin is doing. I'll leave it for now so the sidebar stays marginally functional.